src/optimizeJson.ts

```
@@ -18,6 +18,102 @@ type SourceData = {
+          try {
+            processRecipe(recipe)
+          } catch (err) {
+            // console.warn(`${version} [warn] Removing incorrect recipe: ${err}`)
+            // delete current[i]
+            throw err
+          }
+        }
+        current[name] = current[key]
+      } catch (err) {
+        // console.warn(`${version} [warn] Removing incorrect recipe: ${err}`)
+        throw err
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)